onedal/data/backend/data.h

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************

```

Comment:
Seems like the data module is not a public one
`onedal/data/backend` -> `onedal/backend/data`?

Reply:
I agree - applied